amd_build/hip_files/ATen/CMakeLists.txt.hip

```diff
@@ -0,0 +1,454 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
```

Comment:
Is this something like, the original `CMakeLists.txt` was copied and then edited until it worked with HIP? What's our ongoing maintenance strategy for noting changes to CMakeLists.txt and getting them to HIP? Would it make sense to try to reintegrate these into the real CMakeLists.txt and having search for HIP be controlled by a flag?

Reply:
The CMakeLists.txt files are extremely similar and they already contain a special WITH_ROCM flag (so does the patched setup.py file). They could both be integrated accordingly.